openpype/modules/kitsu/kitsu_module.py

```diff
@@ -124,19 +124,44 @@ def push_to_zou(login, password):
 
 
 @cli_main.command()
-@click.option("-l", "--login", envvar="KITSU_LOGIN", help="Kitsu login")
 @click.option(
-    "-p", "--password", envvar="KITSU_PWD", help="Password for kitsu username"
+    "-prjs",
+    "--projects",
+    envvar="SYNC_PROJECTS",
+    help="Sync specific kitsu projects"
 )
-def sync_service(login, password):
+@click.option(
+    "-l",
+    "--login",
+    envvar="KITSU_LOGIN",
+    help="Kitsu login"
+)
+@click.option(
+    "-p",
+    "--password",
+    envvar="KITSU_PWD",
+    help="Password for kitsu username"
+)
+def sync_service(login, password, projects="^"):
     """Synchronize openpype database from Zou sever database.
 
     Args:
         login (str): Kitsu user login
         password (str): Kitsu user password
+        projects (str): specific kitsu projects
+
+    SYNC_PROJECTS:
+    *: all projects
+    ^: dont sync any project just listen
+    "project01 project02 ...": to choose custom projects
```

Comment:
This `""` syntax is not consistent with others CLI and how `click` behaves. [Multiple option](https://click.palletsprojects.com/en/8.1.x/options/#multiple-options) must be used. I recommend a `-prj` short name option.